<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Đăng nhập nhân viên</string>
  </property>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>294</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tài khoản:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>294</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>109</y>
     <width>294</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng Nhập</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>144</y>
     <width>294</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng Ký</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>57</y>
     <width>294</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mật khẩu:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>294</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
